gprofiler/profilers/java.py

```diff
@@ -417,10 +417,9 @@ def parse_jvm_version(version_string: str) -> JvmVersion:
 
     # version is always in quotes
     _, version_str, _ = lines[0].split('"')
-    build_str = lines[2].split("(build ")[1]
-    assert "," in build_str, f"Didn't find comma in build information: {build_str!r}"
-    # Extra information we don't care about is placed after a comma
-    build_str = build_str[: build_str.find(",")]
+    build_str = lines[1].split("(build ")[1]
+    assert build_str.endswith(")"), f"build information does not end with ): {build_str!r}"
```

Comment:
HS java 8:
```
openjdk version "1.8.0_111"
OpenJDK Runtime Environment (build 1.8.0_111-8u111-b14-2~bpo8+1-b14)
OpenJDK 64-Bit Server VM (build 25.111-b14, mixed mode)
```

if using 2nd line, we need to strip everything after the 2 digits

Reply:
Ah nice, it ends with `-b14` anyway.. so the existing logic holds.